cli/src/cmd/install.rs

```diff
@@ -88,7 +88,8 @@ pub(crate) fn install(
 /// installs the dependency as an ordinary folder instead of a submodule
 fn install_as_folder(dep: &Dependency, libs: &Path) -> eyre::Result<()> {
     let output = Command::new("git")
-        .args(&["clone", &dep.url, &dep.name])
+        // --quiet prevents git from sending info messages to stderr
+        .args(&["clone", "--quiet", &dep.url, &dep.name])
```

Comment:
great!
but weird that the flag comes in between the subcommand and value, can we put this at the end of the array?

Reply:
Neither of these commands picked it up after the positional args. Checking `man git-clone` this is the correct position in this case.

Reply:
FYI, I'm going to remove these flags anyway.